plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/button/index.tsx

```diff
@@ -48,12 +46,11 @@ const featurePluginSupport = {
 		__experimentalSelector:
 			'.wp-block-button.wc-block-components-product-button .wc-block-components-product-button__button',
 	} ),
-	...( typeof __experimentalGetSpacingClassesAndStyles === 'function' &&
-		! isFeaturePluginBuild() && {
-			spacing: {
-				margin: true,
-			},
-		} ),
+	...( typeof __experimentalGetSpacingClassesAndStyles === 'function' && {
```

Comment:
I left these conditionals alone because we didn't really discuss them, and they seem harmless in the sense that we don't *need* to change them as a part of this PR. That said I don't think there has been explicit discussion about stability of these spacing features.

Reply:
Yeah, I don't see any downside to keeping them 👍